waveform_editor/settings.py

```diff
@@ -47,6 +47,17 @@ class NiceSettings(param.Parameterized):
         objects=[INVERSE_MODE, DIRECT_MODE], default=INVERSE_MODE, precedence=-1
     )
     are_required_filled = param.Boolean(precedence=-1)
+    is_direct_mode = param.Boolean(precedence=-1)
+    is_inverse_mode = param.Boolean(precedence=-1)
+
+    @param.depends("mode", watch=True, on_init=True)
+    def set_mode_flags(self):
+        if self.mode == self.INVERSE_MODE:
+            self.is_direct_mode = False
+            self.is_inverse_mode = True
+        else:
+            self.is_direct_mode = True
+            self.is_inverse_mode = False
```

Comment:
While the previous logic is correct, this better conveys intention:

```suggestion
        self.is_direct_mode = self.mode == self.DIRECT_MODE
        self.is_inverse_mode = self.mode == self.INVERSE_MODE
```